js/src/get-started-page/features-card/index.scss

```diff
@@ -1,27 +1,78 @@
 .gla-get-started-features-card {
-	.components-flex {
-		justify-content: space-around;
+	// Adjust <CardHeader> imported from @wordpress/components.
+	// Repeat selector to make it higher priority.
+	.components-card__header.components-card__header {
+		flex-direction: column;
+		gap: $grid-unit-10;
+		padding: $grid-unit-60 $grid-unit * 14.125 $grid-unit-50;
+		text-align: center;
 
 		@media (max-width: $break-small) {
-			flex-direction: column;
+			padding: $grid-unit-30 $grid-unit-30 $grid-unit-20;
+			gap: $grid-unit-20;
 		}
+	}
+
+	.components-flex {
+		gap: 50px;
+		padding: 0 $grid-unit * 11 $grid-unit-60;
 
 		&__block {
 			display: flex;
 			flex-direction: column;
 			align-items: center;
-			max-width: 195px;
-			margin: calc(var(--large-gap) / 4 * 5) 0;
 			text-align: center;
 		}
+
+		@media (max-width: $break-small) {
+			flex-direction: column;
+			gap: 40px;
+			padding: 0 55px $grid-unit-30;
```

Comment:
💅 
From the visual design as below screenshot, looks like the bottom spacing should be larger.

![2022-04-25 13 48 18](https://user-images.githubusercontent.com/17420811/165028215-32396e6b-0ecf-43e2-95d7-c075e96b8988.png)

Reply:
Thanks, updated in 97c9f46.